src/librepgp/stream-armor.cpp

```diff
@@ -979,50 +996,37 @@ rnp_dearmor_source(pgp_source_t *src, pgp_dest_t *dst)
 {
     rnp_result_t res = RNP_ERROR_BAD_FORMAT;
     pgp_source_t armorsrc = {0};
-    uint8_t      readbuf[PGP_INPUT_CACHE_SIZE];
-    size_t       read;
+    {
```

Comment:
Why did you need to put this code into nested `{}` block? It doesn't seem to change anything.

Reply:
@ni4 It releases 32K from the stack.
Without it, the stack should have capacity of 32K+32K = 64K

Reply:
@rrrooommmaaa  But actually, since all bulk reading buffer was moved to rnp_pipe_source(), in this function it is enough to have buffer of `strlen(ST_CLEAR_BEGIN) + 1`, and so no need to add `{}` block.